### DISCUSSION TITLE
GPS Metadata from images

### DISCUSSION THREAD

**Discussion by helifilm-com** [colmap/colmap#3069] [General]
Dear everyone,  I am running COLMAP software trying to recreate  a Stockholm tower using drone footage.  There is embedded GPS info in each of the pictures, so I want to use the spatial analysis. However, COLMAP says “no images with location data”. Below is an example of EXIF-parameters that are embedded in the image files. 

Color Space                     : Uncalibrated
GPS Version ID                  : 2.3.0.0
GPS Latitude Ref                : North
GPS Longitude Ref               : East
GPS Altitude Ref                : Above Sea Level
Image Size                      : 8192x4320
Megapixels                      : 35.4
GPS Altitude                    : 105.8 m Above Sea Level
GPS Latitude                    : 59 deg 19' 37.25" N
GPS Longitude                   : 18 deg 3' 23.88" E
Focal Length                    : 50.0 mm
GPS Position                    : 59 deg 19' 37.25" N, 18 deg 3' 23.88" E

What am  I missing here?  Your help in this would be very appreciated.

Best Regards,

Björn Olin
[helifilm.com](http://helifilm.com/) Archive
https://www.helifilm.com/

**Comment by ahojnnes** [comment]
Could you share at least one or a few of the images for debugging?

**Comment by helifilm-com** [comment]
Hi! I would love to. The DNGs are the originals from an Inspire 3 drone, and the PNGs are my converted images, where I added more of the GPS attributes, to try to make it work. Unfortunately, my images seem to big for the Github chat, but I made some available for download here:

https://we.tl/t-GHdTsXEoNw

**Comment by jared-krauss** [comment]
Did you ever figure this out?

I've been working on a similar issue. I realized that not all of my images had an altitude measurement, even though they had gps coordinates. And so Colmap wasn't picking it up.

THat doesn't seem to be an issue for you. 

I used Exiftool though to change the exif data and update the altitude. 

You could try using exiftool to see what it reads for the exif gps for your image set. In Lightroom all the images had GPS, but their exif..not all of them had altitude. It was weird. 

But once I fixed that, it recognized all GPS coordinates.
